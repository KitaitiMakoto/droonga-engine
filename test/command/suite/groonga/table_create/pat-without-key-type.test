{
  "type": "table_create",
  "dataset": "Default",
  "body": {
    "name"  : "User",
    "flags" : "TABLE_PAT_KEY"
  }
}
