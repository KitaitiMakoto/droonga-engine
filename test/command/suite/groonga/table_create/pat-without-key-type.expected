{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "table_create.result",
  "body": [
    [
      -22,
      0.0,
      0.0,
      "key_type is required for TABLE_PAT_KEY table"
    ],
    false
  ]
}
